src/ia-bookmark-edit.js

```diff
@@ -86,21 +124,37 @@ export class IABookmarkEdit extends LitElement {
     `;
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  get disabledSaveButton() {
+    return html`<button class="save-bookmark" type="submit" value="Save" disabled="disabled">Save</button>`;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  get saveButton() {
+    return html`<button class="save-bookmark" type="submit" value="Save">Save</button>`;
+  }
```

Comment:
Do you need two different save buttons since the only difference is the `disabled` state?